guides/legacy/dependencies.md

```diff
@@ -1,39 +1,30 @@
-### NPM/Yarn and Bower Configuration
+### NPM and Yarn Configuration
 
-Ember CLI supports [NPM](https://www.npmjs.com), [yarn](https://yarnpkg.com/)
-and optionally [Bower](http://bower.io/) for dependency management. It will
-detect whether you are using npm or yarn by the presence of a `yarn.lock` file
-in your project.
-
-A newly generated Ember CLI project only has NPM dependencies, so you will
-notice a `package.json` file at the root of your project, but not a `bower.json`.
-To use Bower packages, you will have to first run `bower init` to create a
-`bower.json` file also at the root of the project.
-
-NPM's `package.json` together with Bower's `bower.json` allow you to declare
-the dependencies of your project.
-Changes to your dependencies should be managed through these files, rather
-than manually installing packages individually.
+Ember CLI supports [NPM](https://www.npmjs.com) and [yarn](https://yarnpkg.com/) 
```

Comment:
small nitpick: `npm` should be lowercase, while `yarn` should by `Yarn` according to their websites 😉

Reply:
Thanks for pointing this out. I noticed their website uses 'npm' lowercased even in the middle of sentences too. I'll go ahead and change all instances of NPM to npm.